### PR TITLE
Updated "miss chance", to "Accuracy"

### PR DESCRIPTION
**Pull request by Masterdiekillsu**
Changed all instances of the usage of the words "miss chance", to "accuracy" in order to reflect the current Combat 2.0 changes.

### CODE REVIEW

**Comment by bleehu** [comment]
Make sure to add labels and assign the reviewers to your issue. that'll make them easier to find.

File: CharacterCreation/03_Feats.txt [hunk 1]
```diff
@@ -921,16 +920,16 @@ Prerequisites:
 *  Hacker or
 *  Knowledge - Nanites at 40
 
-	Add +3 miss chance to a piece of cover for 4 turns.
+	Add -3 accuracy to a piece of cover for 4 turns.
```
Comment (bleehu) [inline CharacterCreation/03_Feats.txt:923]: Scope creep: this feat makes absolutely zero sense and has no flavor text. Who wrote this?

**Review by bleehu — APPROVED** [review]
These all look good! Great work @Masterdiekillsu ! Shipit!

**Comment by Turtlelord26** [comment]
🐢 ❤️
